app.js

```diff
@@ -4,11 +4,14 @@ import Debug from 'debug';
 import logger from 'morgan';
 import cookieParser from 'cookie-parser';
 import bodyParser from 'body-parser';
+import validator from './middleware/validator';
+import session from './middleware/session';
```

Comment:
relative imports (stuff from your own modules) should come after absolute imports (stuff like 'express')